decidim/decidim
# Add direct action to block this participant

#### :tophat: What? Why?
This PR is adding the ability of the admin to block the user directly from interface, as requested in :  #10037 

This PR depends on: #10024

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #?
- Fixes  #10037

#### Testing
1. Login as administrator 
2. Visit the public profile of an user 
3. Click Report User 
4. Observe Block and Hide Checkboxes 
5. Check Block, Hide depending on your test Scenario 
6. Go through the process
7. Observe the user is Blocked 
8. Start the sidekiq runner on queue `user_report`
9. Wait the jobs to be consumed, 
10. Observe the content of the user being hidden 

### :camera: Screenshots
*Please add screenshots of the changes you're proposing*
![image](https://user-images.githubusercontent.com/105683/204085034-3c683c60-cdd7-4096-a7c2-9afad70ac35d.png)

:hearts: Thank you!
